tests/test_transformation/test_yeojohnson_transformer.py

```diff
@@ -84,3 +85,30 @@ def test_non_fitted_error(df_vartypes):
     with pytest.raises(NotFittedError):
         transformer = YeoJohnsonTransformer()
         transformer.transform(df_vartypes)
+
+
+def test_inverse_transform_wit_columns_of_other_types(df_vartypes):
```

Comment:
"wit" has a typo. What do you men by "column of other types"? I don't understand :_(

You mean different data types? as in int and floats?

Reply:
Me neither actually, bad name and mistake by my side. I think I mean to test it with data with a more types like str, int, float and date in this case.

Change its name to 
`test_inverse_transform_automatically_select_only_transformed_columns`